randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -157,7 +157,7 @@ Extra - room_id: [3]
                   Can Use Bombs
                   Can Bounce in Ball and Can Use Any Bombs
               # Trick to break the front speedblocks to leave a tunnel to get to the bomb block without needing a bounce item.
-              Shinespark Tricks (Advanced) and Can Use Any Bombs
+              Shinespark Tricks (Advanced) and Can Use Power Bombs
```

Comment:
I think this should be sufficient for this now. Let me know if you think I've inputted this wrong or if something doesn't make sense.